### DISCUSSION TITLE
Sample images are blurry during FLUX-schnell lora finetune

### DISCUSSION THREAD

**Discussion by laolongboy** [bghira/SimpleTuner#1361] [Q&A]
--attention_mechanism=diffusers
--report_to=tensorboard
--lora_rank=4
--resume_from_checkpoint=latest
--data_backend_config=config/multidatabackend.json
--aspect_bucket_rounding=2
--seed=42
--minimum_image_size=0
--output_dir=outputs
--flux_fast_schedule
--pretrained_model_name_or_path=models--black-forest-labs--FLUX.1-schnell
--pretrained_t5_model_name_or_path=models--stabilityai--stable-diffusion-3.5-large-turbo/text_encoders/t5xxl_fp16.safetensors
--pretrained_vae_model_name_or_path=models--black-forest-labs--FLUX.1-schnell
--lora_type=standard
--lycoris_config=config/lycoris_config.json
--max_train_steps=2000
--num_train_epochs=0
--checkpointing_steps=100
--checkpoints_total_limit=5
--tracker_project_name=lora-training
--tracker_run_name=simpletuner-lora
--model_type=lora
--model_family=flux
--train_batch_size=2
--gradient_checkpointing
--caption_dropout_probability=0.1
--resolution_type=pixel_area
--resolution=1024
--validation_seed=42
--validation_steps=100
--validation_resolution=1024x1024
--validation_guidance=1.0
--validation_guidance_rescale=0.0
--validation_num_inference_steps=4
--user_prompt_library=config/validate_prompts.json
--mixed_precision=bf16
--optimizer=adamw_bf16
--learning_rate=1e-4
--lr_scheduler=polynomial
--lr_warmup_steps=100

I set --flux_fast_schedule=true, validation_guidance=1.0 and validation_num_inference_steps=4. 

Did I miss something need to be specified for Flux-schnell?

step500:

<img width="1297" alt="Image" src="https://github.com/user-attachments/assets/0f8eade6-d66c-4de7-83fa-8fcf601ed78d" />

**Comment by bghira** [comment]
flux schnell isnt really trainable without breaking its distillation first

**Comment by laolongboy** [comment]
> flux schnell isnt really trainable without breaking its distillation first

I see. Is there any recommended de-distilled schnell model?

**Comment by bghira** [comment]
possibly Flex or LibreFlux

**Comment by bghira** [comment]
fixed
